inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_binary_elementwise.cpp

```diff
@@ -39,7 +39,7 @@ void dynamicToStaticShapeBinaryEltwise(std::shared_ptr<ngraph::Node> eltwise) {
     const auto diff = std::abs(lhsRank.get_length() - rhsRank.get_length());
     if (diff) {
         auto & broadcastInput = lhsRank.get_length() < rhsRank.get_length() ? lhsInput : rhsInput;
-        const auto broadcastConst = ngraph::opset3::Constant::create(broadcastInput.get_element_type(), {static_cast<uint64_t>(diff)}, {1});
+        const auto broadcastConst = ngraph::opset3::Constant::create(broadcastInput.get_element_type(), {static_cast<size_t>(diff)}, {1});
```

Comment:
Should it be `int64_t`?

Reply:
class Shape : public std::vector<size_t>, so we should use size_t. I have created a JIRA ticket to to change inheritance to uint64_t

Reply:
But what about *consumer*? As far as I'm concerned, if some operation takes inputs related to shape it should expect that as `int64_t`. @GlebKazantaev could you please confirm?

If it is the case, we have a chance to produce issue here..

Reply:
shape cannot have a negative value. if you look at the line above, you will notice the call of  `std::abs`.

Reply:
Still consumer may interpreter memory in a way it is `int64_t`